cmd/earthly/app/before.go

```diff
@@ -58,10 +58,11 @@ func (app *EarthlyApp) before(cliCtx *cli.Context) error {
 		disableOngoingUpdates := !app.BaseCLI.Flags().Logstream || app.BaseCLI.Flags().InteractiveDebugging
 		forceColor := envutil.IsTrue("FORCE_COLOR")
 		noColor := envutil.IsTrue("NO_COLOR")
+		isGitHubActions := envutil.IsTrue("GITHUB_ACTIONS") || app.BaseCLI.Flags().IsGitHubActions
```

Comment:
I _think_ the flag also handles the case that it's set via the env var (could be wrong though,   I don't recall if the flag is already set at the time of calling `before()`), perhaps worth checking

Reply:
oh nice, will check

Reply:
seems to work